src/airport.js

```diff
@@ -1,6 +1,48 @@
+const Plane = require('./Plane');
+
 class Airport {
-  // here's a starting point for you
+  landedPlanes = [];
+  landedCapacity;
+
+  // constructor(defaultLandedCapacity) {
+  //   this.landedCapacity = defaultLandedCapacity;
+  // }
 
```

Comment:
This was the original constructor before later adding a rejection of negative values and floats. I don't think I could have done that earlier, as it was an edge case which I didn't consider until the end. But in the future I may have that foresight.